EIPS/eip-2681.md

```diff
@@ -23,7 +23,7 @@ Lastly, this facilitates a minor optimisation in clients, because the nonce no l
 
 ## Specification
 
-If `block.number >= FORK_BLOCK` introduce two new restrictions:
+Introduce two new restrictions retroactively from genesis:
 
 1. Consider any transaction invalid, where the nonce exceeds `2^64-1`.
 2. The `CREATE` instruction to abort with an exceptional halt, where the account nonce is `2^64-1`.
```

Comment:
Even if `CREAT2` doesn't use the nonce as an input, does it actually increments it? Need to double check this.

Reply:
@axic very good point. `CREATE2` does increment the nonce. I've updated the EIP.